scripts/onnx/test.sh

```diff
@@ -24,6 +24,17 @@ done
 set -- "${UNKNOWN[@]}" # leave UNKNOWN
 
 pip install pytest scipy hypothesis
+# JIT C++ extensions require ninja.
+pip install ninja --user
+# ninja is installed in /var/lib/jenkins/.local/bin
+export PATH="/var/lib/jenkins/.local/bin:$PATH"
+
+sudo apt-get install libpython-dev
+ls -al /usr/local/include
```

Comment:
Remove these debug command?

Reply:
removed ... from the debug commands it seems the path `/usr/local/include/python2.7` doesn't exist,  that might be the reason why JIT compiler couldn't find `Python.h` file.